oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RecordRequests.jsx

```diff
@@ -33,19 +54,52 @@ export const RecordRequests = ({ record }) => {
         setRequests(sortByStatusCode(response.data?.hits?.hits));
       })
       .catch(error => {
-        console.log(error);
+        setRequestsLoadingError(error);
+      })
+      .finally(() => {
+        setRequestsLoading(false);
       });
+  }, [record.links?.requests]);
+
+  const fetchNewRequests = useCallback(async () => {
+    setRecordLoading(true);
+    setRequestsLoading(true);
+    setRecordLoadingError(null);
+    setRequestsLoadingError(null);
+    return Promise.all([
+      fetchUpdated(record.links?.self, (responseData) => { setRecord(responseData); }),
+      fetchUpdated(record.links?.requests, (responseData) => { setRequests(sortByStatusCode(responseData?.hits?.hits)); })
```

Comment:
We can make the same function (as fetchRecord) named fetchRequests (that also handles its corresponding loading& error states)